modules/search/class.jetpack-search.php

```diff
@@ -1610,7 +1610,17 @@ public function get_filters( WP_Query $query = null ) {
 			} // End foreach().
 		} // End foreach().
 
-		return $aggregation_data;
+		/**
+		 * Modify the filters returned by get_filters()
+		 *
+		 * @module search
```

Comment:
We're passed the code freeze for 6.8.0, so you'll need to update this to `6.9.0` I'm afraid.